device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -92,7 +95,7 @@
     protected ScheduledExecutorService taskScheduler;
     protected IotHubClientState state;
     protected long RECEIVE_PERIOD_MILLIS;
-    protected CustomLogger logger;
+    protected final Logger logger = LoggerFactory.getLogger(DeviceClient.class);
```

Comment:
this.class is not specified and having different loggers and logging configurations for classes in the same hierarchy wont work with this.class